libraries/tests/rtos/mbed/file/main.cpp

```diff
@@ -27,62 +33,64 @@ void sd_thread(void const *argument)
     SDFileSystem sd(p11, p12, p13, p14, "sd");
 #endif
 
-    // Allocate data buffers
-    uint8_t data_written[SIZE] = { 0 };
-    uint8_t data_read[SIZE] = { 0 };
-
     {
         // fill data_written buffer with random data
-        FILE *f = fopen(FILE_NAME, "w");
+        FILE *f = fopen(FILE_NAME, "w+");
         if (f) {
             // write these data into the file
-            printf("Writing %d bytes to file:\r\n", SIZE);
+            printf("Writing %d bytes to file:" NL, SIZE);
             for (int i = 0; i < SIZE; i++) {
                 data_written[i] = rand() % 0xff;
                 fprintf(f, "%c", data_written[i]);
                 printf("%02X ", data_written[i]);
                 if (i && ((i % 20) == 19))
-                    printf("\r\n");
+                    printf(NL);
```

Comment:
Is NL as new line? Is it a macro? I can't find the macro definition.

Isn't this NL host specific? We should use a newline.